tests/core/chain-object/test_contract_call.py

```diff
@@ -0,0 +1,119 @@
+from eth.chains.base import MiningChain
+from tests.core.helpers import (
+    new_transaction,
+)
+
+from eth_utils import (
+    keccak,
+    to_bytes,
+)
+import pytest
+
+
+@pytest.fixture
+def chain(chain_without_block_validation):
+    return chain_without_block_validation
+
+
+@pytest.fixture
+def loaded_simple_contract(chain, funded_address, funded_address_private_key):
```

Comment:
Instead of loading this into the chain by sending a transaction, what if you instead just seeded it in the initial state.  We could alter the `chain_with_block_validation` fixtures to allow injection of extra state data.

```python
# in conftest.py
@pytest.fixture
def base_genesis_state(funded_address):
    return {...}

@pytest.fixture
def genesis_state(base_genesis_state):
    return base_genesis_state

@pytest.fixture
def chain_without_block_validation(genesis_state, ...):
    # we use the `genesis_state` here to seed the initial chain database.

# in another `test_something_about_the_evm.py` file
@pytest.fixture
def genesis_state(base_genesis_state):
    return merge(base_genesis_state, {...<custom-state-things})
```

This allows any test module to define a `genesis_state` fixture and pytest will prioritize the local one to the one in the `conftest.py` file, allowing easy injection of genesis state into those fixtures.